### ISSUE TITLE
eav support in export_records

### ISSUE THREAD

**Issue by yawenl** [redcap-tools/PyCap#82]
I found import_records() have eav type supported. 
It seems like export_records() doesn't have eav supported. Please correct me if I'm wrong.
I'm wondering if I can add support to this,

**Comment by sburns** [comment]
I don't have the REDCap API docs nearby, can you provide what the corresponding cURL command looks like to export records in EAV? It would likely be a small addition to the parameters `Project.export_records` passes along.

**Comment by yawenl** [comment]
Sorry I didn't find cURL example. The following is a RcURL example:
raw_text <- RCurl::postForm(
  uri = redcap_uri
  , token = token
  , content = 'record'
  , format = 'csv'
  , type = 'eav'
  , rawOrLabel = 'raw'
  , exportDataAccessGroups = 'true'
  , .opts = RCurl::curlOptions(ssl.verifypeer = FALSE)
)

**Comment by sburns** [comment]
Yes `type` needs to be passed in the POST parameter, do you feel comfortable making this change and submitting a Pull-Request? Following additions will be necessary:

1. Add a `type` argument in `Project.export_records`. I would default it to whatever the value is "non-EAV" to maintain backwards compatibility.
2. It would be nice to add a small example in `docs/deep.rst` in the section around exporting records.
